### DISCUSSION TITLE
Restore / reset to default destroys version info

### DISCUSSION THREAD

**Discussion by Sagnasty** [rern/rAudio#40] [Fixed Issues]
In an attempt to debug some issues I was having, I attempted to use the "Restore / reset to default" option.
After the system came back from the reset, I noticed that the system / version information had changed (version = "rAudio 0 with no date" if I recall).   Also, from the ADDONS page, the ability to perform rAudio 1 updates was gone.   My only option at that point was to rebuild the system from image (restoring from a previous backup did not solve the issue).

Jim

3 x Raspberry Pi 3B+ Rev 1.3 > HiFiBerry DAC+ Pro
1 x Raspberry Pi 3B Rev 1.2 > HiFiBerry DAC+ Pro
Synology DS718+

**Comment by ceejayemm** [comment]
Jim

This is probably an old problem the affected RA+Rer6 sometimes. For that Rern suggested the following, from an SSH terminal session, to fix it (which it did for me):

echo 20201228 > /srv/http/data/addons/r1

You may then need to run the Addons to pick up the latest changes.

**Reply by Sagnasty** [reply]
Thanks Chris - I will give that a try next time around.

Be advised that this is a repeatable issue, and I reproduced this on two different machines (with rAudio 1 • 202012281).

Jim

3 x Raspberry Pi 3B+ Rev 1.3 > HiFiBerry DAC+ Pro
1 x Raspberry Pi 3B Rev 1.2 > HiFiBerry DAC+ Pro
Synology DS718+

**Comment by rern** [comment]
Configurations in rAudio 1 have been changed from R e6 quite a lot. If restore has broken something, try setup manually.

**Reply by Sagnasty** [reply]
My expectation was that "Reset to default" would bring me back to the point at which the system had been initially booted from image (and then complete the setup manually as you indicated).

The main issue with "**Reset to default**" is that continuing to setup manually is pointless **as I could no longer receive future rAudio 1 updates from the Addon page** ("Reset to default" destroyed that ability along with displayed the missing version info). 

Therefore, the only logical choice at that point was to re-image the micro SD card using the latest image and then proceed to complete the setup manually (an easy and predictable end result).

Hope this helps.

Jim

3 x Raspberry Pi 3B+ Rev 1.3 > HiFiBerry DAC+ Pro
1 x Raspberry Pi 3B Rev 1.2 > HiFiBerry DAC+ Pro
Synology DS718+

**Comment by rern** [comment]
Fixed.
